kit/src/test/java/com/oracle/javafx/scenebuilder/kit/fxom/FXMLPropertiesDisablerVariationsTest.java

```diff
@@ -31,64 +31,40 @@
  */
 package com.oracle.javafx.scenebuilder.kit.fxom;
 
-import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
-import java.util.Arrays;
-import java.util.List;
-
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
-import org.junit.runners.Parameterized.Parameters;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
 
 import com.oracle.javafx.scenebuilder.kit.editor.EditorPlatform.OS;
 
-@RunWith(Parameterized.class)
```

Comment:
Junit5 handles parameterized tests in a very convenient way. No constructor for the test class is needed, also no specialized test runner. All you need is the `@ParameterizedTest` annotation and a value source where I've selected the `CsvCsvSource`. Another advantage is, that we now can place other tests for this class/area in the same test class. 
In Junit4, a seperate test class would have been required.